script/test-template.sh

```diff
@@ -31,6 +31,7 @@ function run_npm_tests() {
 
 function check_errors_in_log() {
     echo; echo -n "--[test ${TEMPLATE}]-- Search for errors in logs output... "
+    if ! [ -f ${LOGFILE} ]; then echo "ERROR Log file ${LOGFILE} not found"; exit 1; fi
```

Comment:
Is there a reason the log file might not have been created?

Reply:
Not in theory. In practice https://github.com/probot/create-probot-app/commit/5178caab367942987725a5fd9ecc979ac882b3b1 introduced a bug where the template name folder was doubled in `LOGFILE` (e.g. from `tmp-tests/templates/basic-js/test.output` to `tmp-tests/templates/basic-js/basic-js/test.output`). 
It's subtle and we didn't notice, hence the new simple test.

Reply:
Ah, got it.